aws/ecs/server_metrics_etl.tf

```diff
@@ -26,6 +26,7 @@ module "masked_server_metrics_etl" {
     environment             = var.env
     bucket_name             = var.masked_metrics_bucket
     metric_token_secret_arn = aws_secretsmanager_secret.metrics_token.id
+    server_events_endpoint  = "https://retrieval.wild-samphire.cdssandbox.xyz/events"
```

Comment:
Could this be a variable to make it easier to share between staging/prod?

Reply:
Yeah it probably should be ... I did mean to do that